lib/CL/pocl_llvm_api.cc

```diff
@@ -854,10 +1022,18 @@ int pocl_llvm_get_kernel_metadata(cl_program program,
                               content.str().size());
 #endif
 
+#if defined(LLVM_OLDER_THAN_3_9)
```

Comment:
Oh ok. Then please rename the *llvm3_9 function to something more descriptive and add function comments to both.